lib/exec.js

```diff
@@ -1,25 +1,15 @@
-var child_exec = require('child_process').exec;
+var child_spawn = require('child_process').spawn;
 
 module.exports = function exec(script) {
 
-	var command_line = script.command;
+	// Grab the command section of the entered command
+	var command = script.command.split(' ')[0];
+	// Anything else is options
+	var options = script.command.split(' ').slice(1);
 
-	Object.keys(script.env).forEach(function (key) {
-	  var value = script.env[key];
+	console.log('to be executed:' + command);
+	var command = child_spawn(command, options, script.env);
```

Comment:
Can you confirm this? As this package was created primarily to support win32 :smile: